shared/wallets/transaction/index.js

```diff
@@ -0,0 +1,178 @@
+      ifFlowErrorsHereItsCauseYouDidntHandleAllActionTypesAbove(props.counterpartyType);
+      */
+      break
+  }
+
+  const amount = <Text type={textTypeSemibold}>{props.amountUser}</Text>
+
+  if (props.counterpartyType === 'wallet') {
+    if (props.yourRole === 'sender') {
+      return (
+        <Text type={textType}>
+          You transferred Lumens worth {amount} from this wallet to {counterparty}.
+        </Text>
+      )
+    }
+
+    return (
+      <Text type={textType}>
+        You transferred Lumens worth {amount} from {counterparty} to this wallet.
+      </Text>
+    )
+  }
+
+  if (props.yourRole === 'sender') {
+    return (
+      <Text type={textType}>
+        You sent Lumens worth {amount} to {counterparty}.
+      </Text>
+    )
+  }
+
+  return (
+    <Text type={textType}>
+      {counterparty} sent you Lumens worth {amount}.
+    </Text>
+  )
+}
+
+type AmountXLMProps = {|
+  yourRole: Role,
+  amountXLM: string,
+  pending: boolean,
+|}
+
+const AmountXLM = (props: AmountXLMProps) => {
+  const color = props.pending
+    ? globalColors.black_20
+    : props.yourRole === 'sender'
+      ? globalColors.red
+      : globalColors.green
+  const amount = `${props.yourRole === 'sender' ? '-' : '+'} ${props.amountXLM}`
+  return (
+    <Text style={{color, textAlign: 'right'}} type="BodyExtrabold">
+      {amount}
+    </Text>
+  )
+}
+
+export type Props = {|
+  ...$Exact<DetailProps>,
+  ...$Exact<CounterpartyIconProps>,
+  // A null timestamp means the transaction is still pending.
+  timestamp: Date | null,
+  amountXLM: string,
+  note: string,
+|}
+
+export const Transaction = (props: Props) => (
+  <Box2 direction="horizontal" fullWidth={true} style={styles.container}>
+    <CounterpartyIcon
+      counterparty={props.counterparty}
+      counterpartyType={props.counterpartyType}
+      large={props.large}
+    />
+    <Box2 direction="vertical" fullHeight={true} fullWidth={true} style={styles.rightContainer}>
+      <Text type="BodySmall">{props.timestamp ? formatTimeForPopup(props.timestamp) : 'Pending'}</Text>
+      <Detail
+        large={props.large}
+        yourRole={props.yourRole}
+        counterparty={props.counterparty}
+        counterpartyType={props.counterpartyType}
+        amountUser={props.amountUser}
+      />
+      {props.large && (
+        <Text style={styles.note} type="Body">
+          {props.note}
+        </Text>
+      )}
+      <AmountXLM pending={!props.timestamp} yourRole={props.yourRole} amountXLM={props.amountXLM} />
+    </Box2>
+  </Box2>
+)
+
+const styles = styleSheetCreate({
+  container: {
+    padding: globalMargins.tiny,
+    paddingRight: globalMargins.small,
+  },
+  note: platformStyles({
+    // TODO: Consider using markdown quoting.
+    common: {
+      marginTop: globalMargins.xtiny,
+    },
+    isElectron: {borderLeft: `3px solid ${globalColors.lightGrey2}`, paddingLeft: 8},
```

Comment:
if you specify borderLeftColor borderLeftWidth electron should also just work with that

Reply:
Ah, done. Tried it before, forgot I had to specify `borderLeftStyle` for electron

Reply:
You can do borderStyle and it'll work on both.

Reply:
Done